release_notes_generator/generator.py

```
@@ -25,7 +25,7 @@
 
 from github import Github
 
-from release_notes_generator.filter import Filter
+from release_notes_generator.filter import Filter, FilterByRelease
```

Ruff (0.12.2)

`release_notes_generator.filter.Filter` imported but unused

Remove unused import: `release_notes_generator.filter.Filter`

(F401)